backend/tests/v5/unit/models/jobs.test.js

```diff
@@ -37,6 +37,31 @@ const testGetJobsToUsers = () => {
 	});
 };
 
+const testAddDefaultJobs = () => {
+	describe('Add default jobs', () => {
+		test('should add the default jobs', async () => {
+			const DEFAULT_JOBS = [
+				{ _id: 'Client', color: '#a6cee3' },
+				{ _id: 'Architect', color: '#213f99' },
+				{ _id: 'Structural Engineer', color: '#33a02c' },
+				{ _id: 'MEP Engineer', color: '#fb9a99' },
+				{ _id: 'Project Manager', color: '#e31a1c' },
+				{ _id: 'Quantity Surveyor', color: '#ff7f00' },
+				{ _id: 'Asset Manager', color: '#ffff99' },
+				{ _id: 'Main Contractor', color: '#b15928' },
+				{ _id: 'Supplier', color: '#6a3d9a' },
+			];
+
+			const fn = jest.spyOn(db, 'insertMany').mockImplementation(() => {});
+			await Jobs.addDefaultJobs('teamspace');
+			expect(fn.mock.calls.length).toBe(1);
```

Comment:
Again, can we start using the syntax with `shouldBeCalled` etc , as mentioned from a previous PR https://github.com/3drepo/3drepo.io/pull/3091#discussion_r819654394

Typically a lot cleaner with less lines.